nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-bootstrap.js

```diff
@@ -339,6 +342,7 @@
                     nfActions.init();
                     nfQueueListing.init();
                     nfComponentState.init();
+                    nfComponentVersion.init(nfSettings, nfProcessGroupConfiguration);
```

Comment:
Should be able to remove the injection of the nfProcessGroupConfiguration module here.